app/workers/functions.php

```diff
@@ -484,17 +487,7 @@ public function execute(string $trigger, string $projectId, string $executionId,
             ->setParam('userId', $userId)
             ->setParam('webhooks', $webhooks)
             ->setParam('event', 'functions.executions.update')
-            ->setParam('eventData', [
-                '$id' => $execution['$id'],
-                'functionId' => $execution['functionId'],
-                'dateCreated' => $execution['dateCreated'],
-                'trigger' => $execution['trigger'],
-                'status' => $execution['status'],
-                'exitCode' => $execution['exitCode'],
-                'stdout' => $execution['stdout'],
-                'stderr' => $execution['stderr'],
-                'time' => $execution['time']
-            ]);
+            ->setParam('eventData', $execution->getArrayCopy());
```

Comment:
Any reason for this change? are we sure we didn't want to hide any internal attribute?

Reply:
Fixed it by using the Execution Response as a filter 👍